recipes/caniusepython3/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "caniusepython3" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 22a38ef229b621919988e5c2787a1d524c0283f343d7e2923858a8dfa424c0b0
+
+build:
+  number: 0
+  entry_points:
+    - caniusepython3 = caniusepython3.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - backports.functools_lru_cache
+    - distlib
+    - futures  # [py2k]
```

Comment:
There is a futures package for py3, so this selector can be removed and then the whole recipe made noarch I think.

Reply:
I had it like that originally but `futures` only has Python 2.7 binaries:
https://anaconda.org/conda-forge/futures/files

Reply:
ohhhh shit you are right. 🤦‍♂

Reply:
I read "future" instead of "futures". Grrrrr.